### PR TITLE
live-reload-server-config - implements #2

### PR DESCRIPTION
**Pull request by Kesse8401**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,17 +1,77 @@
...
+
 It suppose to:
 - enable  two players to have a game
 - display information whose turn is is
 - mark tiles with "X" or "0"
 - acknowledge end of the game by getting one of the winning combinations
 - display the result/winner
 
-2. Technology
+You can find a copy of the project here: https://github.com/Kesse8401/TicTacToe
```
Comment (bartlomn) [inline README.md]: Maslo maslane - przeciez sa wlasnie na tej stronie - delete

File: README.md [hunk 2]
```diff
@@ -1,17 +1,77 @@
...
+
+
+Prerequisites
+
+/* What things you need to install the software and how to install them */
```
Comment (bartlomn) [inline README.md]: good call, you can list stuff like
* node (and the version you currently have)
* npm and its version

File: README.md [hunk 3]
```diff
@@ -1,17 +1,77 @@
...
+
+Live-server is required to start the project. You can find it here: https://www.npmjs.com/package/live-server along with the instructions regarding installing
```
Comment (bartlomn) [inline README.md]: Bad call. This whole section is incorrect.
`live-server` is now part of your dependencies. That's the whole point. Anybody using your code just needs to:
1. Clone the repo
2. `npm install`
3. `npm start`

That's it. - Put the above here instead.

File: README.md [hunk 4]
```diff
@@ -1,17 +1,77 @@
...
+
+You can install live-server locally by navigating you your target file and adding this command to your terminal:
+
+npm install live-server --saveDev
+
+
+
+!!! Installing
+
+/* A step by step series of examples that tell you how to get a development env running */
```
Comment (bartlomn) [inline README.md]: This is incomplete. No incomplete code/comment belongs on `master`

File: README.md [hunk 5]
```diff
@@ -1,17 +1,77 @@
...
-3. Tests
+Running the tests
 
 This project does not include any tests, as it structure is fairly simple and including tests would unnecessarily overcomplicate it.
-Tests would be added further down the line, as my understanding of the ecosystem improves.
\ No newline at end of file
+Tests would be added further down the line, as my understanding of the ecosystem improves.
+
+
+Deployment
```
Comment (bartlomn) [inline README.md]: remove this section - we've not covered it, it's not even part of this task. Focus!

File: README.md [hunk 6]
```diff
@@ -1,17 +1,77 @@
...
+/* Add additional notes about how to deploy this on a live system */
+
+
+Built With
+
+This project is being created with the use of ES6, plain JavaScript and Node.js. I chose plain JavaScript as it allows me to implement all necessary elements mentioned above, and helps me to achieve deeper understanding of the language.
+I used one of npm module- live-server in order to work with HotReload
+
+
+Versioning
```
Comment (bartlomn) [inline README.md]: Remove, irrelevant

File: index.html [hunk 7]
```diff
@@ -8,6 +8,7 @@
 
  <body>
   <p>Hello World! You are grey and sleepy today.</p>
+  <p>Terminal doesn't work as expected (under my command)! Hovever at least I created package.json and changed license...</p>
```
Comment (bartlomn) [inline index.html:11]: Good for you 👍

File: package.json [hunk 8]
```diff
@@ -0,0 +1,22 @@
+{
```
Comment (bartlomn) [inline package.json]: Looking good 👍

**Review by bartlomn — CHANGES_REQUESTED** [review]
Minor changes requested, mainly regarding the readme content. Otherwise we're making progress.

**Comment by bartlomn** [comment]
please also rename the PR title to include reference to issue #2 (use hash character followed by 2)

File: package.json [hunk 9]
```diff
@@ -0,0 +1,22 @@
...
+    "type": "git",
+    "url": "git+https://github.com/Kesse8401/TicTacToe.git"
+  },
+  "author": "Agnes Nowak",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/Kesse8401/TicTacToe/issues"
+  },
+  "homepage": "https://github.com/Kesse8401/TicTacToe#readme",
+  "dependencies": {
```
Comment (bartlomn) [inline package.json]: Ah, almost missed this one.
You were supposed to add a dev dependency. Change `dependencies` to `devDependencies`.
I will ask you at home about the difference between the two.

File: README.md (hunk 4 above)
Comment (Kesse8401) [reply README.md]: Idk this one. Are those steps that I have completed? Or that somebody has to? Haven't we covered that already? (Download/clone , install, start) ?

File: package.json (hunk 9 above)
Comment (Kesse8401) [reply package.json]: All about one (mentioned above in the comments are done)

File: README.md [hunk 10]
```diff
@@ -1,17 +1,69 @@
-1. Project info
-This project is about creating simple Tic Tac Toe game.
+
+TicTacToeGit
+
+Simple game of Tic Tac Toe 
+
+Getting Started
+
+/*These instructions will get you a copy of the project up and running on your local machine for development and testing purposes. See deployment for notes on how to deploy the project on a live system.*/
```
Comment (bartlomn) [inline README.md]: Sorry, didn't spot it earlier.
This is markdown, not a js file. I assume using the comment notation here is a mistake.
Please [see this](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet) for more details on the syntax

File: README.md [hunk 11]
```diff
@@ -1,17 +1,69 @@
...
 
-2. Technology
 
-This project is being created with the use of ES6 and plain JavaScript. I chose plain JavaScript as it allows me to implement all necessary elements mentioned above, and helps me to achieve deeper understanding of the language.
 
-3. Tests
+
+Prerequisites
+
+/* What things you need to install the software and how to install them */
```
Comment (bartlomn) [inline README.md]: use markdown syntax

File: README.md [hunk 12]
```diff
@@ -1,17 +1,69 @@
...
+
+!!!node (and the version you currently have)
```
Comment (bartlomn) [inline README.md]: So which version is it?

File: README.md (hunk 4 above)
Comment (bartlomn) [reply README.md]: this bit:
```
Say what the step will be
+
+Give the example
+And repeat
```
seems to be a bit bullshit

File: README.md [hunk 13]
```diff
@@ -1,17 +1,69 @@
...
+
+- Clone the repository
+- npm install
+- npm start
+
+
+
+!!! Installing
+
+/* A step by step series of examples that tell you how to get a development env running */
```
Comment (bartlomn) [inline README.md]: Also, pls use markdown syntax

**Review by bartlomn — CHANGES_REQUESTED** [review]
Some more minor things. Seems you've confused markdown with JavaScript

**Comment by bartlomn** [comment]
Ah, noticed one more thing - quite a big fuckup tbh. One of your commits in this PR adds `node_modules` directory to the git repo. This wrong on many levels - pls fix ASAP.
Steps to fix:
* remove the directory - try to use terminal
* add relevant entry to gitignore
* `npm install` your project to have npm download `node_modules` for you again
* verify `node_modules` not in the changeset
* commit

**Review by bartlomn — APPROVED** [review]
Looking good 👍 
Merge and delete the branch

**Pull request merged**